.gitignore

```diff
@@ -1,3 +1,6 @@
+.gitignore
```

Comment:
`.gitignore` should absolutely not be ignored, otherwise it can't be committed to the repo.